src/lib/DALMeteorMethods/GeneralDBMethods.js

```diff
@@ -0,0 +1,23 @@
+/*eslint-disable*/
```

Comment:
Don't make generic/general utility files, it encourages 'laziness' and people will just start throwing whatever they want in here. Making it a huge clusterf**k
Rename file to exactly what it's for, something like UserMethod.js